tools/lightning/filter.md

```diff
@@ -0,0 +1,122 @@
+---
+title: TiDB-Lightning Table Filter
+summary: Use black and white lists to filter out tables, ignoring them during import.
+category: tools
+---
+
+# TiDB-Lightning Table Filter
+
+TiDB-Lightning supports setting up black and white lists to ignore certain databases and tables. This can be used to skip cache tables, or manually partitions the data source on a shared storage to allow multiple Lightning instances work together without interfering each other.
+
+The filtering rule is similar to MySQL `replication-rules-db`/`replication-rules-table`.
+
+## Filtering databases
+
+```toml
+[black-white-list]
+do-dbs = ["pattern1", "pattern2", "pattern3"]
+ignore-dbs = ["pattern4", "pattern5"]
+```
+
+If the name of a database matched *any* pattern in the `do-dbs` array in the `[black-white-list]` section, the database will be included.
```

Comment:
- For these three sentences, we can use a bulleted list.
- In documentation, try to avoid using the word "will". Use the general present tense instead. Please also update the following part.